src/api/ipfs.actions.ts

```diff
@@ -0,0 +1,23 @@
+import { create as createIpfsClient } from 'ipfs-http-client'
+
+import { throwError } from 'util/errors'
+
+const client = createIpfsClient({
+  url: 'https://ipfs.io',
```

Comment:
considering having this live in a config / env variable, but seems overkill for now